packages/lib/src/interactions/SelectionTool.tsx

```diff
@@ -36,18 +36,21 @@ function SelectionTool(props: Props) {
 
   const camera = useThree((state) => state.camera);
   const { dataToWorld, worldToData } = useVisCanvasContext();
+
+  const [startPoint, setStartPoint] = useState<Vector2>();
+  const [selection, setSelection] = useRafState<Selection | undefined>(
```

Comment:
I now store the entire `selection` instead of just the end point. I know this means the `startPoint` is technically stored twice, but I don't think it's a problem. Especially since, for #1296, I'm gonna need to store more than just the start point in data space.

Reply:
Would it be possible to work with the `startPoint` in `selection` to remove the `startPoint` state ?